docs/PowerStack.md

```diff
@@ -2,47 +2,123 @@
+## Block Diagram
 
-## Block Diagram / External Connections
+<img src="images/amdc-power-stack.svg" />
 
-Pictorial representation of power stack interface, with blocks and descriptions of what they do.
-- Show where 0R resistors are for disconnecting power etc
-- Jumper selections do WHAT?
-- Status lines?
+The full power stack interface block diagram is shown above. It is compacted into 8x duplicated channels for easy representation. The connectors and jumpers depicted *outside* of the blue box are not replicated on the PCB.
 
-## Connector Pinout
+Each DB-15 connector is designed to control a single three-phase two-level inverter. This includes six PWM signals and four status signals. The user can configure the voltage for both the status and PWM signals independently. The user can also supply each inverter with a power supply designed to run the gate drivers that is distributed through the AMDC; this reduces wire routing overall. Finally, all PWM outputs are only enabled when the E-STOP input is disabled. If the E-STOP input becomes open-circuit, all PWM signals go to a low state, thus turning off all switches.
 
-Explaination and list of pinout
+<img src="images/amdc-power-stack-highlighted.jpg" width="75%" />
 
-Picture of all 8 power stack connectors with arrows which correspond to firmware names etc
+<img src="images/amdc-input-screw-terminals.jpg" width="75%" />
 
-## Schematics
+### 1. PWM Signals
 
-Discussion related to schematics for power stack interface...
+The PWM gate drive signals are generated by the PicoZed FPGA at 1.8V. These are then level-shifted up to the user-supplied `VDRIVE` voltage. Each PWM signal then goes to the DB-15 connector. Each PWM signal is ANDed with the E-STOP signal. Thus, when the E-STOP is opened, the PWM signals go to a logic low state.
 
-## PCB Layout
+### 2. Status Signals
+
+Each inverter has four status lines (labeled A, B, C, and D). These signals operate at the `VDD` voltage level, which is configurable by the jumper on the PCB. Each status line (A/B/C/D) can be configured as either an input or output, depending on the jumper selection. Note that all eight inverters use the same directionality for the status signals (i.e. if status signal A is an input, it is an input for all eight inverters).
+
+These status signals can be used as arbitrary I/O between the AMDC and the power stack. Typically, they are used to read faults states, reset the power stack, read ready states from the power stack gate drives, etc. The voltage level which is selected by the jumper on the AMDC is supplied to each DB-15 connector so the power stack can use the correct I/O voltage level for communication.
+
+#### Jumpers
+
+On the AMDC, the following are the jumpers used to set status line voltage level and direction:
+
+- JP1: Status voltage level
+- JP4: Status A direction
+- JP5: Status B direction
+- JP6: Status C direction
+- JP7: Status D direction
+
+### 3. Voltage Rails
```

Comment:
Should we be providing a link to the [AMDC Power Distribution](https://github.com/Severson-Group/AMDC-Hardware/blob/develop/docs/PowerDistribution.md) page?

Reply:
@elsevers How is this? https://github.com/Severson-Group/AMDC-Hardware/commit/460f9645724f0426789cf446568bdcc03c724036

Reply:
Yep, looks good.